docker-hub/repos.md

```diff
@@ -53,6 +53,35 @@ images to this repository.
 
 3. Enter the name of the repository to confirm the deletion and click **Delete**.
 
+## Consolidating a repository
+
+### Personal to personal
+
+When consolidating personal repositories, you can pull private images from the initial repository and push them into another repository that is owned by you. To avoid losing your private images, perform the following steps listed below.
+
+1. Navigate to Docker Hub, create a Docker ID and select the personal subscription.
+2. Using `docker login` from the CLI, log in using your original Docker ID and pull your private images.
+3. Tag your private images with your newly created Docker ID using:
+`docker tag namespace1/docker101tutorial ‘new_namespace/docker101tutorial`
+4. Using `docker login` from the CLI, log in with your newly created Docker ID, and push your newly tagged private images to your new Docker ID namespace.
+`docker push <new_namespace>/docker101tutorial`
```

Comment:
Nitpick, not a blocker. 

Some examples have `<>` and some don't. 
For example `docker push <new_namespace>/docker101tutorial` has it and  `docker tag namespace1/docker101tutorial new_namespace/docker101tutorial` does not have it.

Suggest to add `<>` or remove `<>` from all for consistency.

Reply:
Thanks Craig! Removed the <> from the examples